graylog2-web-interface/src/views/components/searchbar/QueryInput.jsx

```diff
@@ -12,19 +14,20 @@ import SearchBarAutoCompletions from './SearchBarAutocompletions';
 import type { Completer } from './SearchBarAutocompletions';
 
 type Props = {|
-  disabled: boolean,
-  value: string,
-  completers: Array<Completer>,
   completerFactory: (Array<Completer>) => AutoCompleter,
+  completers: Array<Completer>,
+  disabled: boolean,
   onBlur?: (string) => void,
   onChange: (string) => Promise<string>,
   onExecute: (string) => void,
   placeholder: string,
+  theme: ThemeInterface,
```

Comment:
Could we add `theme: themePropTypes.isRequired` to the `propTypes` object as well?